docs/integrations-guides/rollup-guides/orbit/deployment.md

```diff
@@ -34,7 +36,11 @@ The script will take a few minutes to complete as it prints out the addresses of
 
 ## How to deploy a Rollup on Testnet using our UI
 
-While you can interact with the deployed Rollup creator directly, we recommend using the [EigenDA x Orbit chain deployment portal](https://orbit.eigenda.xyz/) to deploy a rollup for a friendly UX and easy-to-use configs. Currently, the only supported testnets are Ethereum Holesky and Arbitrum Sepolia.
+While you can interact with the deployed Rollup creator directly, we recommend using our [chain deployment portal](https://orbit.eigenda.xyz/) to deploy a rollup for a friendlier devx and easy-to-use configs. Currently, the only supported testnets are:
```

Comment:
Is the chain deployment portal something we are scoping to support for Optimism as well?

Initial thought is to specify Orbit x EigenDA

Reply:
[4b309db](https://github.com/Layr-Labs/eigenda-docs/pull/37/commits/4b309db8d48325e0051f222632aad8fea8a70d45)